heron_description/launch/description.launch

```diff
@@ -11,15 +12,16 @@
     <group if="$(eval namespace == '')">
       <param name="robot_description"
              command="$(find heron_description)/scripts/env_run $(find heron_description)/urdf/configs/$(arg config)
-                      $(find xacro)/xacro '$(find heron_description)/urdf/heron.urdf.xacro'
-                      debug:=$(arg hydro_debug)" />
+                      $(find xacro)/xacro.py '$(find heron_description)/urdf/heron.urdf.xacro'
```

Comment:
Why add the `.py`?

Reply:
Copy-and-paste error on my part.  Fixed.